rdt-open-reader/src/testCovidIndonesia/assets/DeviceDefinition.json

```diff
@@ -0,0 +1,2518 @@
+{
```

Comment:
is this different from the other DeviceDefinition?

If so, how? If not, why do we need it?

Reply:
It shouldn't be too different. The initial idea was to isolate the tests from changes in the `DeviceDefinition` JSON i.e. they shouldn't break because of schema changes in the `DeviceDefinition`. Rather, they should only break as a result of invalid queries on any valid `DeviceDefinition`.

It was easier to reason about it that way i.e. once sure that the queries are correct, worry about any issues arising as a result of invalid `DeviceDefinition` JSON. I guess ideally we'd want the tests to break in either situation - if the schema changes which may render the queries invalid or if the queries are invalid given a valid schema.

Reply:
Also, if we think of the feature as eventually supporting sync of FHIR Resources from the server, we might still need static versions of valid Resources in the test folder. This might also necessitate updating the static Resources each time changes are made to their schema.

Reply:
Those are both very good points, you've convinced me that keeping it separate and static as you have done makes sense

Reply:
Overall, this does sound like a weakness in the testing that we may see problems with, e.g. that a change in the a DeviceDefinition may cause errors we won't be able to detect until it's run, and since they're meant to be dynamic (as you've pointed out) we there will be untestable unknown here. Later this could be a reasonable place for a integration test checking it can load whatever DeviceDefinitions happen to be static in the app or dynamic from the server, not sure, we'd need to think that through more

Reply:
Yea, agreed.